shelley/chain-and-ledger/executable-spec/src/Cardano/Ledger/Era.hs

```diff
@@ -5,17 +5,21 @@
 module Cardano.Ledger.Era
   ( Era,
     Crypto,
+    ValueType,
   )
 where
 
 import qualified Cardano.Ledger.Crypto as CryptoClass
 import Data.Kind (Type)
 import Data.Typeable (Typeable)
+import qualified Shelley.Spec.Ledger.Val as ValClass
```

Comment:
General comment: module structure is useful to guide having clean dependencies. Here we're saying that something which is very general (has been put in `Cardano.Ledger`) is depending on something declared within a specific ledger. 

Of course, arguably `Val` is general enough, but in such a case the argument should be for moving it to the `Cardano.Ledger` namespace. We should certainly avoid such dependencies.

Reply:
I will move Val in a next PR